app/src/main/java/Extensions.kt

```diff
@@ -57,4 +58,10 @@ fun MenuItem.loadIconFromUrl(context: Context, imageUrl: String) {
                     icon = circularIcon
                 }
             })
+}
+
+inline fun SharedPreferences.edit(f: SharedPreferences.Editor.() -> Unit) {
+    val editor = edit()
+    editor.f()
+    editor.apply()
```

Comment:
This needs to be `commit()`, since I let RxJava handle the threading. Maybe add a parameter to switch between `apply()` and `commit()`?